packages/demo/src/components/App/index.js

```diff
@@ -14,13 +14,21 @@ const TaglineRe = /^(.*)\[Cerebral\](.*)$/
 export default connect(
   {
     t: translations,
-    loggedIn: 'user.$loggedIn'
+    loggedIn: 'user.$loggedIn',
+    $loading: 'app.$loading',
```

Comment:
My convention for $something is just for storage to indicate stuff that is not saved (pure UI). I think we should not use the $ in props name (or use it every time)...